FND/Places/Addresses.rdf

```diff
@@ -99,29 +120,457 @@
 		<skos:definition>system for allocating addresses to objects</skos:definition>
 	</owl:Class>
 	
+	<owl:NamedIndividual rdf:about="&fibo-fnd-plc-adr;Apartment">
+		<rdf:type rdf:resource="&fibo-fnd-plc-adr;SecondaryUnitDesignator"/>
+		<rdfs:label>apartment</rdfs:label>
+		<skos:definition>room or a group of related rooms, among similar sets in one building, designed for use as a dwelling</skos:definition>
+		<fibo-fnd-plc-adr:requiresSecondaryUnitRange rdf:datatype="&xsd;boolean">true</fibo-fnd-plc-adr:requiresSecondaryUnitRange>
```

Comment:
SO - I have a bunch of individuals that reuse this definition in this style from the postal service addressing standard.  They are used for business addresses to generate bar codes / QCR codes as well.  I'd be happy to show those to you.  There is some precision here that WF needs for their addressing tables, and that Retail is also interested in.  If people don't want to use that they can use the address lines.

Reply:
The alternative is to use an address line, simply text.

Reply:
No, surely the alternative in OWL is to have a restriction on the class Apartment, not this horrible boolean.

Reply:
It's an individual, not a class, so that doesn't work - there are many more kinds of secondary units in the US extension - these I kept here because they were in the original informative ontology.  I could remove them altogether and keep them with the US extension if you prefer.